test/test_devices.cpp

```
@@ -112,22 +113,23 @@ TEST(cubeb, enumerate_devices)
   r = cubeb_init(&ctx, "Cubeb audio test", NULL);
   if (r != CUBEB_OK) {
     fprintf(stderr, "Error initializing cubeb library\n");
-    ASSERT_EQ(r, CUBEB_OK);
+    return r;
```

It'd be better to assert everywhere rather than returning a result and asserting in the caller, since you lose the location of the failure this way.